R/wflow_use_github.R

```diff
@@ -134,6 +134,7 @@
 #' }
 #'
 #' @importFrom httpuv startServer
+#' @importFrom httr RETRY
```

Comment:
Please remove the roxygen tag because you explicitly use the namespace `httr::`

Reply:
oh yeah, no problem

Reply:
fixed in https://github.com/jdblischak/workflowr/pull/199/commits/99036e145858ea2b9fb66749c571a270a5179e7f